folio/reports/missing_items.sql

```diff
@@ -0,0 +1,40 @@
+-- metadb:function missing_items
```

Comment:
There should not be a space after the `--`.  I will fix this and also adjust the SQL formatting.